solace-spring-cloud-stream-binder-opentelemetry/solace-spring-cloud-stream-binder-instrumentation/pom.xml

```diff
@@ -0,0 +1,124 @@
+  <description>A Spring Cloud Stream Solace Binder Instrumentation for use with OpenTelemetry Java Agent</description>
+  <url>
+    https://github.com/${repoName}/solace-spring-cloud/tree/${project.scm.tag}/solace-spring-cloud-stream-binder-opentelemetry
+  </url>
+
+  <properties>
+    <opentelemetry.version>1.48.0</opentelemetry.version>
+    <opentelemetry-javaagent-extension.version>2.14.0-alpha</opentelemetry-javaagent-extension.version>
+    <auto-service.version>1.0.1</auto-service.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>${opentelemetry.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry.javaagent</groupId>
+      <artifactId>opentelemetry-javaagent-extension-api</artifactId>
+      <version>${opentelemetry-javaagent-extension.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auto.service</groupId>
+      <artifactId>auto-service-annotations</artifactId>
+      <version>${auto-service.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy</artifactId>
+      <version>1.14.17</version>
+    </dependency>
+
+    <!-- Dependencies needed for JCSMP auto-instrumentation -->
+    <dependency>
+      <groupId>com.solacesystems</groupId>
+      <artifactId>sol-jcsmp</artifactId>
+      <version>${solace.jcsmp.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.solace</groupId>
+      <artifactId>solace-opentelemetry-jcsmp-integration</artifactId>
+      <version>${sol-jcsmp-otel-integration.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <!-- Dependencies needed for Solace Binder auto-instrumentation -->
+    <dependency>
+      <groupId>com.solace.spring.cloud</groupId>
+      <artifactId>spring-cloud-stream-binder-solace</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.14.0</version>
+        <configuration>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>com.google.auto.service</groupId>
+              <artifactId>auto-service</artifactId>
+              <version>${auto-service.version}</version>
+            </path>
+          </annotationProcessorPaths>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.4.2</version>
+        <configuration>
+          <archive>
+            <manifestFile>src/main/resources/META-INF/MANIFEST.MF</manifestFile>
```

Comment:
instead of overriding the manifest file completely, would it be better to add your custom attributes on top of the existing generated manifest?

https://maven.apache.org/shared/maven-archiver/examples/manifestEntries.html